src/platform/silabs/rs911x/wfx_sl_ble_init.cpp

```diff
@@ -57,9 +55,10 @@ void SilabsBleWrapper::rsi_ble_on_mtu_event(rsi_ble_event_mtu_t * rsi_ble_mtu)
  */
 void SilabsBleWrapper::rsi_ble_on_gatt_write_event(uint16_t event_id, rsi_ble_event_write_t * rsi_ble_write)
 {
-    bleEvent.eventType           = BleEventType::RSI_BLE_GATT_WRITE_EVENT;
-    bleEvent.eventData->event_id = event_id;
-    memcpy(&bleEvent.eventData->rsi_ble_write, rsi_ble_write, sizeof(rsi_ble_event_write_t));
+    SilabsBleWrapper::BleEvent_t bleEvent;
+    bleEvent.eventType          = BleEventType::RSI_BLE_GATT_WRITE_EVENT;
+    bleEvent.eventData.event_id = event_id;
+    memcpy(&bleEvent.eventData.rsi_ble_write, rsi_ble_write, sizeof(rsi_ble_event_write_t));

```

Comment:
Was it intended for this event to stai as is?

Reply:
Yeah, was seeing new issues pop up during commissioning. Keeping this as it is for now. will be modifying and fixing this in followup PR. Since it is only for refactor

Reply:
Done